docs/how_to_guides/distributing_oci_layouts.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Distributing OCI Layouts
+sidebar_position: 6
+---
+
+# Distributing OCI Layouts
+
```

Comment:
Could you add just a brief introduction to the OCI Image Layout and the purpose of this doc? It will help readers and users understand the value and scenario of this doc.

Reply:
@FeynmanZhou added. Please let me know if any more changes are needed.

Reply:
Unblocking for merge.